examples/skyline_selection_results.py

```diff
@@ -0,0 +1,114 @@
+import numpy as np
+import pandas as pd
+import os, pathlib, itertools
+import warnings
+
+warnings.simplefilter('ignore')
+os.chdir('..')
+
+from fp.traindata_samplers import CompleteData
+from fp.missingvalue_handlers import CompleteCaseAnalysis
+from fp.dataset_experiments import GermanCreditDatasetSexExperiment
+from fp.scalers import NamedStandardScaler
+from fp.learners import LogisticRegression, DecisionTree
+from fp.post_processors import NoPostProcessing, RejectOptionPostProcessing
+from fp.pre_processors import NoPreProcessing, DIRemover
+
+import matplotlib.pyplot as plt
+import seaborn as sns
+
+# creating list of parameters that we will alter to observe variations
+seeds = [0xbeef, 0xcafe, 0xdead, 0xdeadcafe]
+learners = [LogisticRegression()]
+
+processors = [(NoPreProcessing(), NoPostProcessing()), (DIRemover(1.0), NoPostProcessing()), (NoPreProcessing(), RejectOptionPostProcessing())]
+
+
+skyline_order = ['accuracy', 'selection_rate', 'false_discovery_rate']
```

Comment:
most users might not know what a skyline is, can we give these variables a different name?

Reply:
Fixed by renaming the variables and adding explanations for readability.